less/about/preferences.less

```diff
@@ -455,6 +455,61 @@ table.sortableTable {
   overflow-y: scroll;
 }
 
+.modal .dialog.paymentHistory .sectionTitle {
+  text-align: left;
+}
+
+span.paymentHistoryButton.browserButton {
+  display: block;
+  color: @braveOrange;
+  font-size: 14px;
+  white-space: nowrap;
+}
+
+.paymentHistoryFooter span.okButton.primaryButton {
+  background-color: gray;
+  float: right;
+  margin-top: -10px;
+  display: inline;
+  padding-top: 5px;
+  padding-left: 35px;
+  padding-right: 35px;
+  padding-bottom: 5px;
+}
+
+div.nextPaymentSubmission {
+  display: inline;
+}
+.nextPaymentSubmission span {
+  font-size: 12px;
+  display: inline;
+}
+
+.modal .dialog.paymentHistory {
+  padding: 0px;
+
+  .dialog-header {
+    background-color: #EEE;
+    height: 20px;
+    padding-bottom: 10px;
+  }
+
+  .dialog-body {
```

Comment:
Your recommendation fixed this: https://github.com/brave/browser-laptop/pull/3391/commits/2a729d9aacc94127d7bddb5338449cd915235f8c#diff-4b50ff14dadc9756719a577a72f520d6R500